src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -607,30 +608,34 @@ public bool TryFormat(Span<char> destination, out int charsWritten, [StringSynta
         /// <returns><paramref name="value" /> converted to its nearest representable half-precision floating-point value.</returns>
         public static explicit operator Half(float value)
         {
-            const int SingleMaxExponent = 0xFF;
-
-            uint floatInt = BitConverter.SingleToUInt32Bits(value);
-            bool sign = (floatInt & float.SignMask) >> float.SignShift != 0;
-            int exp = (int)(floatInt & float.BiasedExponentMask) >> float.BiasedExponentShift;
-            uint sig = floatInt & float.TrailingSignificandMask;
-
-            if (exp == SingleMaxExponent)
-            {
-                if (sig != 0) // NaN
-                {
-                    return CreateHalfNaN(sign, (ulong)sig << 41); // Shift the significand bits to the left end
-                }
-                return sign ? NegativeInfinity : PositiveInfinity;
-            }
-
-            uint sigHalf = sig >> 9 | ((sig & 0x1FFU) != 0 ? 1U : 0U); // RightShiftJam
-
-            if ((exp | (int)sigHalf) == 0)
-            {
-                return new Half(sign, 0, 0);
-            }
-
-            return new Half(RoundPackToHalf(sign, (short)(exp - 0x71), (ushort)(sigHalf | 0x4000)));
+            var v0 = Vector128.CreateScalarUnsafe(0x3880_0000u); //Minimum exponent for rounding
+            var v1 = Vector128.CreateScalarUnsafe(0x3800_0000u); //Exponent displacement #1
+            var v2 = Vector128.CreateScalarUnsafe(0x8000_0000u); //Sign bit
+            var v3 = Vector128.CreateScalarUnsafe(0x7f80_0000u); //Exponent mask
+            var v4 = Vector128.CreateScalarUnsafe(0x0680_0000u); //Exponent displacement #2
+            var v5 = Vector128.CreateScalarUnsafe(65520.0f);     //Maximum value that is not Infinity in Half
+            var v = BitConverter.SingleToUInt32Bits(value);
+            var vval = Vector128.CreateScalarUnsafe(value);
+            vval = (vval.AsUInt32() & ~v2).AsSingle();  //Clear sign bit
```

Comment:
Just do `vval = Vector128.Abs(vval)`. We take care of doing this efficiently.

If you need to do this for other reasons, like reusing a hoisted constant (which I wouldn't expect you need to do, as we should already be cse'ing the implicit `-0.0` constant with any local `-0.0` constant), you'd want to explicitly use `Vector128.AndNot` instead and leave a comment explaining why.